roles/setup_pgd/tasks/pgd_commitscope_configuration.yml

```diff
@@ -0,0 +1,158 @@
+- name: Configure PGD Cluster subgroups
+  block:
+    - name: Create PGD node sub group with two nodes
+      ansible.builtin.include_role:
+        name: manage_dbserver
+        tasks_from: execute_sql_scripts
+      vars:
+        pg_query:
+          - query: "SELECT bdr.create_node_group(node_group_name := '{{ pgd_two_node_group_name }}',
+                                          parent_group_name := '{{ pgd_cluster_name }}',
+                                          join_node_group := false
+                                          )"
+            db: "{{ pgd_cluster_database }}"
+        ignore_query_execution_error: true
+      when:
+        - inventory_hostname == pgd_cluster_nodes[0].inventory_hostname or inventory_hostname == pgd_cluster_nodes[0].inventory_hostname        
+        - configure_pgd_camo|bool or configure_pgd_groupcommit|bool
+        - _pgd_two_node_group_exists|int == 0
+
+    - name: Switch primary node towards PGD node sub group with two nodes
+      ansible.builtin.include_role:
+        name: manage_dbserver
+        tasks_from: execute_sql_scripts
+      vars:
+        pg_query:
+          - query: "SELECT bdr.switch_node_group(node_group_name := '{{ pgd_two_node_group_name }}',
+                                          wait_for_completion := true
+                                          )"
+            db: "{{ pgd_cluster_database }}"
+        ignore_query_execution_error: true
+      when:
+        - inventory_hostname == pgd_cluster_nodes[0].inventory_hostname or inventory_hostname == pgd_cluster_nodes[0].inventory_hostname
+        - configure_pgd_camo|bool or configure_pgd_groupcommit|bool
+        - _pgd_two_node_group_exists|int == 0
+
+    - name: Switch secondary node towards PGD node sub group with two nodes
+      ansible.builtin.include_role:
+        name: manage_dbserver
+        tasks_from: execute_sql_scripts
+      vars:
+        pg_query:      
+          - query: "SELECT bdr.switch_node_group(node_group_name := '{{ pgd_two_node_group_name }}',
+                                          wait_for_completion := true
+                                          )"
+            db: "{{ pgd_cluster_database }}"
+        ignore_query_execution_error: true
+      when:
+        - inventory_hostname == pgd_cluster_nodes[1].inventory_hostname
+        - configure_pgd_camo|bool or configure_pgd_groupcommit|bool
+        - _pgd_two_node_group_exists|int == 0        
+
+- name: Configure PGD Cluster with a CAMO Commit Scope
+  ansible.builtin.include_role:
+    name: manage_dbserver
+    tasks_from: execute_sql_scripts
+    apply:
+      throttle: 1
+  vars:
+    pg_query:
+      - query: "SELECT bdr.add_commit_scope(commit_scope_name := '{{ pgd_camo_scope_name }}',
+                                          origin_node_group := '{{ pgd_two_node_group_name }}',
+                                          rule := 'ALL ( {{ pgd_two_node_group_name }} ) ON durable CAMO DEGRADE ON (timeout=500s) TO ASYNC'
```

Comment:
Had the same thoughts with this SQL statement.

This is a larger discussion because adding the support for the other options will impact the deadline, not to mention we do not know if these other ON options are buggy or working at all.

Reply:
I think we should take the time and implement comprehensively and thoroughly, especially since this is just one revision of the `setup_pgd` role and more need to be done, I think we should avoid implementing something that we know needs more work done to make dynamic. If it means pushing a deadline back a day or two to check the implementation works, I think that is better than having to go back in a few months to implement multiple parameters and have to re-familiarize yourself with commit scopes/PGD/the `setup_pgd` role.

Reply:
i see from the documentation (https://www.enterprisedb.com/docs/pgd/latest/durability/commit-scopes/) that `visible` is the default, when not specified.  i think it makes sense to have a variable set to `visible` by default, and always explicitly set the confirmation level, where we want to be able to set it to `durable` for any specific testing we're doing.  e.g. setting up CAMO